times_reader/transforms.py

```diff
@@ -23,6 +23,20 @@
     "CSet_CD",
 }
 
+attr_com_def = {
```

Comment:
Could we have a docstring for this please?

Reply:
Yes, will do. Basically this specifies defaults, so ideally should be in some separate file...